src/common/providers/identity.ts

```diff
@@ -52,7 +52,8 @@ const CLAIMS_MAX_PAYLOAD_SIZE = 1000;
 
 /**
  * Shorthand auth blocking events from GCIP.
- * @internal
+ * @hidden
```

Comment:
Note: Internal _should_ have been fine, but API Extractor was crashing so I changed it.